src/component/NavBar.js

```diff
@@ -1,27 +1,61 @@
-import React from "react";
-import { Nav, NavItems, NavLinks, NavTitle } from "../Styles/StyledComponents";
+import React, { Component } from "react";
+import {
+  Nav,
+  NavItems,
+  NavLinks,
+  NavTitle,
+  MenuIcon
+} from "../Styles/StyledComponents";
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
+import { faAlignJustify } from "@fortawesome/free-solid-svg-icons";
+import NavMobileMenu from "./NavMobileMenu";
+import Backdrop from "./Backdrop";
 
-const NavBar = () => (
-  <Nav>
-    <NavTitle>CodePurpose</NavTitle>
-    <NavItems>
-      <li>
-        <NavLinks href="/">Home</NavLinks>
-      </li>
-      <li>
-        <NavLinks href="/">About</NavLinks>
-      </li>
-      <li>
-        <NavLinks href="/">Blog</NavLinks>
-      </li>
-      <li>
-        <NavLinks href="/">Careers</NavLinks>
-      </li>
-      <li>
-        <NavLinks href="/">Contact</NavLinks>
-      </li>
-    </NavItems>
-  </Nav>
-);
+class NavBar extends Component {
+  state = { mobileMenuOpen: false };
+
+  menuToggleHandler = () => {
+    this.setState(prevState => {
+      return { mobileMenuOpen: !prevState.mobileMenuOpen };
+    });
+  };
+
+  backdropHandler = () => {
+    this.setState({ mobileMenuOpen: false });
+  };
+
+  render() {
+    const { mobileMenuOpen } = this.state;
+    return (
+      <div>
+        <Nav>
+          <NavTitle>CodePurpose</NavTitle>
+          <NavItems>
+            <li>
+              <NavLinks href="/">Home</NavLinks>
+            </li>
+            <li>
+              <NavLinks href="/">About</NavLinks>
+            </li>
```

Comment:
![image](https://user-images.githubusercontent.com/1214752/58742818-885b2400-83e3-11e9-856b-0d6e4dc52520.png)
The navbar is getting pushed off-screen and I can't navigate to it.

Reply:
I haven't been able to recreate this error seems to work fine on my dev tools.